### PR TITLE
Building Quality Assistance + Coaching with Levels 1-4

### PR DESCRIPTION
**Pull request by michaelnoonan**
This is a significant reworking to identify the different value I am looking to gain from Quality Coaches contributing from Level 1-4.

# How to review

1. As a quality coach, is this guidance practical and concrete?
1. Would I be happy holding myself and my teammates to account?
1. Is the differentiation between L2-4 sensible and align with the principles we used for software-engineers?
1. Are there any glaring gaps in the value we want from quality coaches?

### CODE REVIEW

**Review by mudra-octopus — APPROVED** [review]

File: Quality-Assistance.md [hunk 1]
```diff
@@ -2,34 +2,95 @@
...
+- I confidently organise, coordinate, and run quality coaching activities, like QA KickOff and QA Demo, and how to reap the most value from these activities.
+
+# Lead Quality Coach (Level 4)
+
+> Building on Senior Quality Coach (Level 3), I am a force multiplier, deliberately shifting my focus from individual quality coaching toward uplifting the whole product delivery team.
+
+## I am an expert in our customers and our business and could be an effective product manager or customer advocate
+
+- I have a broad and thorough understanding of the problems our customers are trying to solve and how we do/don't solve them, helping me make good tradeoffs with the entire picture in mind.
+- In addition to our product delivery teams, I am well-connected to our other teams, like support, customer advisory, trust and security, and customer success.
```
Comment (mudra-octopus) [inline Quality-Assistance.md:82]: Isn't `Customer success team` now `Customer advisory team`?

Comment (michaelnoonan) [reply Quality-Assistance.md:82]: Yes, and `Sales` is `Customer Success`, I think.

File: Quality-Assistance.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+## I am actively improving our entire product delivery team
+
+- I have a clear picture of the ideal product delivery team and help our teams make appropriate and timely improvements each cycle.
+- I help our team clearly identify negative quality trends and help make appropriate and timely improvements each cycle.
+- I raise appropriate and timely concerns in #product-delivery and contribute effectively to conversations that help clarify ways we can improve.
+
+## I am actively improving our quality coaching team
+
+- I have a clear picture of the ideal quality coaching team and help make appropriate and timely improvements each cycle.
```
Comment (mudra-octopus) [inline Quality-Assistance.md:94]: Elaborating on 
`help make appropriate and timely improvements each cycle.`
- I experiment with different quality coaching activities to identify the practices that will provide the most confidence in delivery each cycle.

File: Quality-Assistance.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+## I elevate product delivery teams
+
+- Product development teams genuinely want my assistance with their work.
+- I raise the level of coordination, collaboration, and clarity in the teams I'm coaching.
+- My written and verbal communication is clear, concise, articulate, and persuasive.
+- My contributions to team discussions are timely, net-positive, informative, and helpful.
+
+## I am an advocate for our customers and technical quality
+
+- I am easily recognised as an advocate for technical quality at Octopus.
```
Comment (mudra-octopus) [inline Quality-Assistance.md:25]: I am easily recognised as an advocate for technical and product quality at Octopus

**Comment by mudra-octopus** [comment]
This is a great write-up ❤️ . Thanks a lot of taking time out and putting it together. It will provide a lot of guidance to me and new quality coaches.

File: Quality-Assistance.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
+## Quality Assistance
```
Comment (Waldo000000) [inline Quality-Assistance.md:1]: Here and in [Confluence](https://octopushq.atlassian.net/wiki/spaces/IN/pages/892370955/Quality+Assistance+Team), there's a mixture of "Quality Assistance engineer", "Quality Assistance", "quality assurance", and "quality coaching". Do they all mean different things, or are there some synonyms we can eliminate?

File: Quality-Assistance.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+## I am an egoless, team-oriented person
+
+- I actively build good relationships with individuals in our product delivery teams, including product managers, product designers, and software engineers.
+- I can manage difficult conversations and tactfully challenge others, leaving them with a positive impression of myself and Octopus as a whole.
+- Even when I disagree with something, I commit to the agreed course of action, work enthusiastically toward the best outcome, and learn from the experience.
+- I recognize and accept that we will always be learning and improving how we work, and my approach allows me to tactfully challenge the status quo or confidently adapt to change.
+
+## I elevate product delivery teams
+
+- Product development teams genuinely want my assistance with their work.
```
Comment (Waldo000000) [inline Quality-Assistance.md:18]: 1. Is a "product delivery team" the same as a "product development team"?
2. I assume that a quality coach is embedded _within_ such team(s), but the way this is phrased implies they are on the outside looking in. If my assumption is correct, consider a tweak:

```suggestion
- The people that I'm working with in product delivery teams genuinely want my assistance with their work.
```

File: Quality-Assistance.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+- I raise the level of coordination, collaboration, and clarity in the teams I'm coaching.
```
Comment (Waldo000000) [inline Quality-Assistance.md:19]:
```suggestion
- I raise the standard of coordination, collaboration, and clarity in the teams I'm coaching.
```
(to clarify that by "raise the level of" we do not necessarily mean "raise the _amount_ of")

File: Quality-Assistance.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+- My written and verbal communication is clear, concise, articulate, and persuasive.
+- My contributions to team discussions are timely, net-positive, informative, and helpful.
+
+## I am an advocate for our customers and technical quality
+
+- I am easily recognised as an advocate for technical and product quality at Octopus.
+- I am easily recognised as an advocate for things that actually matter to our customers.
```
Comment (Waldo000000) [inline Quality-Assistance.md:26]: > I am easily recognised as an advocate for things that actually matter to our customers.

I like this as a grounding principle ❤️

**Review by Waldo000000 — COMMENTED** [review]
Nice! I left a couple of minor comments for your consideration.

File: Quality-Assistance.md (hunk 2 above)
Comment (michaelnoonan) [reply Quality-Assistance.md:94]:
```suggestion
- I help the quality coaching team improve how they do daily work so we can achieve better outcomes with the teams we coach.
```

File: Quality-Assistance.md (hunk 4 above)
Comment (michaelnoonan) [reply Quality-Assistance.md:1]: Good question. As I've talked to people in other scenarios, they talk about the "practice of quality assistance" as a model and "quality coach" as a role.

We don't have to follow that convention. Perhaps we remove everything except "quality coach" and derivatives? What do you think @Waldo000000

Comment (mudra-octopus) [reply Quality-Assistance.md:1]: I think so too. `Quality coaches` provide `quality assistance` in general. 
➕  on

**Comment by michaelnoonan** [comment]
I think I've addressed everything including Confluence pages. Good enough to merge?

**Review by Waldo000000 — APPROVED** [review]

**Review by mudra-octopus — APPROVED** [review]

**Comment by mudra-octopus** [comment]
LGTM 👍

**Pull request merged**